openedx_caliper_tracking/utils.py

```diff
@@ -102,3 +102,27 @@ def get_certificate_url(user_id, course_id):
         ))
     )
     return certificate_uri
+
+
+def replace_value_strings_in_json_data(old_string, new_string, json_data):
```

Comment:
Please use a better (and smaller) method name

Reply:
@danialmalik any suggestions?
I picked this one as it is quite descriptive.

Reply:
replace_strings_in_dict